minique/models/queue.py

```diff
@@ -30,14 +30,8 @@ def clear(self) -> Any:
         """
         return self.redis.delete(self.redis_key)
 
-    def enqueue_initial(self, job: "Job", payload: dict) -> None:  # type: ignore[type-arg]
-        assert payload["queue"] == self.name
-        with self.redis.pipeline() as p:
-            p.hset(job.redis_key, mapping=payload)
-            if payload["job_ttl"] > 0:
-                p.expire(job.redis_key, payload["job_ttl"])
-            p.rpush(self.redis_key, job.id)
-            p.execute()
+    def enqueue(self, job: "Job") -> None:
+        self.redis.rpush(self.redis_key, job.id)
```

Comment:
If we're getting rid of the `enqueue_initial` API, then we can probably also not have the new `enqueue()` API either, so the `rpush` could be moved to the pipeline in `_define_and_store_job`, so that internal function'd get a new kwarg `enqueue: bool`?

Reply:
did cross my mind; was thinking that and an user might want to call `queue.enqueue` manually in some corner-cases :thinking:

Reply:
but we can remove it for now and add later if needed

Reply:
as there is now two arguments that queueing up needs (proper payload with the queue name and the `enqueue: bool`) I switched to whole thinkg for `queue_name: str` as we have no other payload modifications at this moment :shrug: